app/controllers/searches_controller.rb

```diff
@@ -4,7 +4,25 @@ class SearchesController < ApplicationController
   # GET /searches
   # GET /searches.json
   def index
-    @searches = Search.all.order(score: :desc).page(params[:page]).per(params[:per_page])
+    @searches = Search.all.order(
```

Comment:
Do we need any additional indexing to handle this?

Reply:
it was fast enough with 4 records in my index 🤷‍♂️